src/styles/mo-components/_fixed-width.scss

```diff
@@ -0,0 +1,9 @@
+.full-page-width { 
+    width: 100vw;
+    position: relative;
+    left: calc(-50vw + 50%);
+}
+.max-width-695 {
```

Comment:
This is very specific, can we put the width in `_variables.css` and give a semantic name to the class?

Reply:
This is a good flag. I just chatted with Bri and even though this value seems pretty arbitrary and specific, we think that is appropriate for a new form style like the one we're implementing. I definitely agree it should be encoded in a variable, though, and I'll make that change.

- [ ] Set width of inner div in full width form as variable, give semantic name to class